.github/workflows/main.yml

```
@@ -28,8 +28,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v2
-      - name: Install molecule
-        run: pip install molecule
+      - name: Install molecule and Ansible
+        run: pip install molecule git+https://github.com/ansible/ansible.git@stable-2.10
```

Here I am just picking the current highest stable version. And letting FAM take the lead on how we craft Ansible github workflows (i.e. https://github.com/theforeman/foreman-ansible-modules/blob/develop/.github/workflows/main.yml#L68)

---

Can't you use this:
```suggestion
        run: pip install molecule ansible~=2.10.0
```

My reading of https://www.python.org/dev/peps/pep-0440/#compatible-release is that you'll get the latest 2.10 release from pypi. This avoids resorting to a git checkout.

---

That works for me. @evgeni any reason to go one route or the other from your experience?

---

Well, `ansible` on PyPI is not `ansible` in git, starting with 2.10.
When you install from git, you really get `ansible-base` (or `ansible-core`, whatever), whereas from PyPI you also get all the collections.

Given we don't *actually* care for the ansible version here, you can just fetch `ansible` for PyPI, which will give you 3.0 (with ansible-base 2.10) and everything should work?